src/node.rs

```diff
@@ -1635,10 +1630,12 @@ impl<S: Send + Sync + 'static + ForkSource + std::fmt::Debug> EthNamespaceT for
         not_implemented("mining")
     }
 
-    fn send_transaction(
```

Comment:
Well darn... hardhat functions use `send_transaction` (e.g. `eth_sendTransaction`) under the hood for things like `hardhat_impersonateAccount` 😞 

Just something to keep in mind as we move forward